Stylesheets/Components/input.css

```diff
@@ -0,0 +1,93 @@
+.input-box {
+    padding: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    color: var(--color-primary);
+}
+
+.input-box input,
+.input-box textarea {
+    font-size: 1.2rem;
+    padding: 1rem;
+    border: 2px solid var(--color-primary);
+    border-radius: var(--container-border);
+    background-color: var(--color-tertiary-variant2);
```

Comment:
from what i have seen in video, this is not blending with the background of body. Try to change it to lighter shade of grey (lighter than body's background)

Reply:
thankyou. I've rectified the color now
Please let me now if it is correct now

![image](https://user-images.githubusercontent.com/90134948/152506616-bb7da73a-8764-497d-8150-5867a5a40d2f.png)

Reply:
Looks good now